src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfFeature.java

```diff
@@ -0,0 +1,1320 @@
+
+        // Pseudogene owing to a SNP/DIP but in other individuals/haplotypes/strains the gene is translated.
+        polymorphic_pseudogene,
+
+        // Pseudogene owing to a reverse transcribed and re-inserted sequence.
+        retrotransposed,
+
+        // Pseudogene where protein homology or genomic structure indicates a pseudogene, but the presence of locus-specific transcripts indicates expression.
+        transcribed_processed_pseudogene,
+        transcribed_unprocessed_pseudogene,
+        transcribed_unitary_pseudogene,
+
+        // Pseudogene that has mass spec data suggesting that it is also translated.
+        translated_processed_pseudogene,
+        translated_unprocessed_pseudogene,
+
+        // A species specific unprocessed pseudogene without a parent gene, as it has an active orthologue in another species.
+        unitary_pseudogene,
+
+        // Pseudogene that can contain introns since produced by gene duplication.
+        unprocessed_pseudogene,
+
+        // Used to tag mistakes in the public databases (Ensembl/SwissProt/Trembl)
+        artifact,
+
+        // Long, intervening noncoding (linc) RNA that can be found in evolutionarily conserved, intergenic regions.
+        lincRNA,
+
+        // Unspliced lncRNA that is several kb in size.
+        macro_lncRNA,
+
+        // Transcript where ditag and/or published experimental data strongly supports the existence of short non-coding transcripts transcribed from the 3'UTR.
+        three_prime_overlapping_ncRNA,
+
+        // Otherwise viable coding region omitted from this alternatively spliced transcript because the splice variation affects a region coding for a protein domain.
+        disrupted_domain,
+
+        // Short non coding RNA gene that forms part of the vault ribonucleoprotein complex.
+        vaultRNA,
+
+        // A non-coding locus that originates from within the promoter region of a protein-coding gene, with transcription proceeding in the opposite direction on the other strand.
+        bidirectional_promoter_lncRNA;
+
+        public static GeneTranscriptType getEnum(String s) {
+
+            if (s.startsWith("3")) {
+                s = "three_" + s.substring(1);
+            }
+
+            // Looks like sometimes RNA is spelled `rna`.
+            // Here's a fix for that:
+            s = s.replace( "rna", "RNA" );
+
+            return GeneTranscriptType.valueOf(s);
+        }
+
+        @Override
+        public String toString() {
+            String s = super.toString();
+
+            if ( s.startsWith("three_") ) {
+                s = "3" + s.substring(7);
+            }
+            return s;
+        }
+
+    }
+
+    public enum GeneTranscriptStatus {
+        KNOWN,
+        NOVEL,
+        PUTATIVE
+    }
+
+    public enum LocusLevel {
+        // Verified locus
+        ONE("1"),
+        // Manually annotated locus
+        TWO("2"),
+        // Automatically annotated locus
+        THREE("3");
```

Comment:
I pulled the names directly from what the field contents were.  I'll refactor them into meaningful names.